### DISCUSSION TITLE
Dedicated 'white' and 'black' theme color options

### DISCUSSION THREAD

**Discussion by JordanZimmitti** [saadeghi/daisyui#2770] [Ideas / feature requests]
Hi Everyone,

I'm not sure if there is a workaround for this but I have two Daisy UI themes, one light and one dark.

For my light theme, I use 
```
'neutral': '#333333',
'neutral-content': '#F7F9FA',
'base-100': '#F7F9FA',
'base-content': '#333333',
```

For my dark theme, I use
```
'neutral': '#CFCFCF',
'neutral-content': '#333333',
'base-100': '#333333',
'base-content': '#CFCFCF',
```

There are some occasions where I want a daisy-ui component color to stay white or black agnostic of the theme that is set. Two dedicated color options such as a `white` and a `black` color options would be very helpful

I followed the tutorial here: https://daisyui.com/blog/how-to-add-new-colors-to-daisyui/
but this does not add the new color options to the daisy-ui components like btn-white

If this is currently possible, if someone can point me in the right direction on how to do this I would really appreciate it.

Thanks!

**Comment by saadeghi** [comment]
There are 2 ways
- Adding a `data-theme` attribute to that specific element (or section) you want to stay untouched by the theme.
  ```
  <button class="btn" data-theme="light">This button will always use light theme colors</button>
  ```
- Using Tailwind CSS utility classes to change the default colors
  ```
  <button class="btn bg-black hover:bg-gray-800 text-white">Always black button</button>
  ```

Let me know if you have a question.
